modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/viewmodel/PodcastViewModel.kt

```diff
@@ -364,6 +368,120 @@ class PodcastViewModel
         }
     }
 
+    fun multiSelectSelectNone() {
+        val uiState = uiState.value as? UiState.Loaded ?: return
+        when (uiState.showTab) {
+            PodcastTab.EPISODES -> multiSelectEpisodesHelper.deselectAllInList(uiState.episodes)
+            PodcastTab.BOOKMARKS -> multiSelectBookmarksHelper.deselectAllInList(uiState.bookmarks)
+        }
+    }
+
+    fun <T> multiSelectAllUp(multiSelectable: T) {
```

Comment:
I added these multi-select methods with generics to the view model. Maybe it needs a better place. I'm open to suggestions.